src/aap_eda/settings/default.py

```diff
@@ -434,3 +437,15 @@ def _get_secret_key() -> str:
     "ansible_base.authentication.backend.AnsibleBaseAuth",
     "django.contrib.auth.backends.ModelBackend",
 ]
+ANSIBLE_BASE_FEATURES = {
+    "AUTHENTICATION": True,
+    "FILTERING": False,
+    "SWAGGER": False,
+}
```

Comment:
FYI, this will change with the split.

Reply:
I initially wanted to include the split changes in this PR but I thought I would do in another PR after since this is more urgent to unblock QE pipelines.